acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -52,13 +53,18 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.ThreadContext;
 import org.junit.After;
 import org.junit.Rule;
 import org.junit.rules.TestName;
 import org.junit.rules.TestWatcher;
 import org.junit.runner.Description;
 
 public class AcceptanceTestBase {
+  static {
+    System.setProperty("log4j2.isThreadContextMapInheritable", "true");
```

Comment:
Doesn't this Log4j property need setting before the Log4j2 framework get initialised? (isn't it too late by here)

Reply:
Yes, it does, and no, it isn't, since it works ;). The framework is initialized when anything from it is used, and the property is set here earlier than that.
Though to tell the truth I can't find now anything explicitly sanctioning this way of doing things.

Anyway, I guess this is fragile in that some code change elsewhere might cause the "early enough" assumption to fail. So if preferred this could be set in a log4j2.property file. I just thought that here it's closer to its purpose.

Reply:
Yeah put it a property file somewhere.

You'll want to also explicitly set 
```
disableThreadContextMap = false
disableThreadContext = false
```

As if either of those are set to true by environment variables, the code isn't going to work
https://logging.apache.org/log4j/2.x/manual/thread-context.html
>Set the system property `disableThreadContextMap` to `true` to disable the Thread Context Map.
Set the system property `disableThreadContext` to `true` to disable both the Thread Context Map and Stack.

Reply:
[Environment variables have higher priority than property files](https://logging.apache.org/log4j/2.x/manual/configuration.html#System_Properties) anyway, so setting those in a property file will not help for this scenario.

Do we really want to force log4j2's behavior like that anyway? If so, then I guess the only way to do so is to set the values manually once the self-configuration is finished. That sounds gory to me. 

IMO, if the user has set environment variables like that, they already know enough to take care of the situation by themselves.